spec/ngsild_swagger.json

```diff
@@ -0,0 +1,1077 @@
+                    "application/json"
+                ],
+                "responses": {
+                    "204": {
+                       "description": "No Content. The context source registration was successfully updated"
+                    },
+                    "400": {
+                        "description": "Bad Request",
+                        "schema": {
+                            "$ref": "ngsild_schema.json#/definitions/ApiErrorResponse"
+                        }
+                    },
+                    "404": {
+                        "description": "Not Found",
+                        "schema": {
+                            "$ref": "ngsild_schema.json#/definitions/ApiErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions": {
+            "post": {
+                "description": "Creates a new Subscription within an NGSI-LD system",
+                "operationId": "createSubscription",
+                "parameters": [
+                    {
+                        "name": "SubscriptionDescription",
+                        "description": "Payload body in the request contains a JSON-LD object which represents the Subscription that is to be created",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "ngsild_subscriptions_schema.json#/definitions/SubscriptionDescription"
+                        }
+                    }
+                ],
+                "consumes": [
+                    "application/json"
+                ],
+                "responses": {
+                    "201": {
+                        "description": "Created",
+                        "headers": {
+                            "Location" : {
+                                "schema": {
+                                    "type": "string"
+                                },
+                                "description": "Contains the resource URI of the created Entity resource"
+                            }
+                        }
+                    },
+                    "400": {
+                        "description": "Bad Request",
+                        "schema": {
+                            "$ref": "ngsild_schema.json#/definitions/ApiErrorResponse"
+                        }
+                    },
+                    "409": {
+                        "description": "Already exists",
+                        "schema": {
+                            "$ref": "ngsild_schema.json#/definitions/ApiErrorResponse"
+                        }
+                    }
+                }
+            },
+            "get": {
+                "description": "Retrieves the subscriptions available in an NGSI-LD system",
+                "operationId": "retrieveSubscriptions",
+                "parameters": [
+                    {
```

Comment:
this was dropped

Reply:
I will check this deeper. Right now I think that this corresponds with the 5.11.5